commit/tokenprice/observation.go

```diff
@@ -81,17 +80,10 @@ func (p *processor) ObserveFeedTokenPrices(ctx context.Context, lggr logger.Logg
 	if err != nil {
```

Comment:
There is a sort 3 lines up. It's no longer needed because you're returning a map instead of a slice.